Automattic/wp-calypso
# Marketplace: Order categories by search popularity

#### Proposed Changes

* Order marketplace plugin categories by search popularity

#### Testing Instructions

* Order of marketplace categories should be listed according to search popularity seen in pdh6GB-1kr-p2#comment-2200
* Had to guess with events / crm which are new categories from https://github.com/Automattic/wp-calypso/pull/65812.

![Screenshot 2022-07-25 at 17-44-37 Plugins ‹ test — WordPress com](https://user-images.githubusercontent.com/811776/180725854-89d107b5-cb97-4a6a-bebc-38ffb18d94a1.png)


#### Pre-merge Checklist






Fixes https://github.com/Automattic/wp-calypso/issues/65219
